tests/arrows/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -908,7 +908,10 @@ foo(c, a => b, d)
 foo(a => b, d)
 
 =====================================output=====================================
-promise.then(result => result, err => err);
+promise.then(
+  result => result,
+  err => err
+);
```

Comment:
I afraid it will be annoy many

Reply:
Hehe, this is one of the cases I think became better with this PR 😆

Reply:
Typically, most `promise.then` calls are long enough that they’ll be split on multiple lines anyways.